driver/pad_constant_driver.hpp

```diff
@@ -0,0 +1,508 @@
+
+    return miopenStatusSuccess;
+}
+
+template <typename Tgpu, typename Tref>
+int ConstantPadDriver<Tgpu, Tref>::RunForwardCPU()
+{
+    mloConstantPadForwardRunHost(
+        inputDesc, outputDesc, input.data(), output_host.data(), padding, value);
+    return miopenStatusSuccess;
+}
+
+template <typename Tgpu, typename Tref>
+int ConstantPadDriver<Tgpu, Tref>::VerifyForward()
+{
+    RunForwardCPU();
+
+    // for (int i = 0; i < output.size(); i++)
+    //     printf("output[%d] = %f\n", i, output[i]);
+
+    for(int i = 0; i < output.size(); i++)
+    {
+        if(output[i] != output_host[i])
+        {
+            std::cout << "output[" << i << "] = " << output[i] << " != "
+                      << "output_host[" << i << "] = " << output_host[i] << std::endl;
+            std::cout << "ConstantPadDriver: Forward verification failed." << std::endl;
+            return -1;
+        }
+    }
+    std::cout << "ConstantPadDriver: Forward verification passed." << std::endl;
+    return miopenStatusSuccess;
+}
+
+template <typename Tgpu, typename Tref>
+int ConstantPadDriver<Tgpu, Tref>::RunBackwardCPU()
+{
+    mloConstantPadBackwardRunHost(
+        backwardOutputDesc, outputDesc, backward_output_host.data(), output.data(), padding);
+    return miopenStatusSuccess;
+}
+
+template <typename Tgpu, typename Tref>
+int ConstantPadDriver<Tgpu, Tref>::RunBackwardGPU()
+{
+    float kernel_total_time = 0;
+    float kernel_first_time = 0;
+
+    Timer t;
+    START_TIME
+
+    for(int i = 0; i < inflags.GetValueInt("iter"); i++)
+    {
+        miopenPadConstantBwd(GetHandle(),
+                             backwardOutputDesc,
+                             outputDesc,
+                             backward_output_dev->GetMem(),
+                             out_dev->GetMem(),
+                             padding.data(),
+                             padding.size());
+
+        float time = 0.0;
+        miopenGetKernelTime(GetHandle(), &time);
+        kernel_total_time += time;
+        if(i == 0)
+            kernel_first_time = time;
+    }
+
+    if(inflags.GetValueInt("time") == 1)
+    {
+        STOP_TIME
+        int iter = inflags.GetValueInt("iter");
+
+        if(WALL_CLOCK)
+            std::cout << "Wall-clock Backward Time Elapsed: " << t.gettime_ms() / iter << " ms"
+                      << std::endl;
+
+        float kernel_avg_time =
+            iter > 1 ? (kernel_total_time - kernel_first_time) / (iter - 1) : kernel_first_time;
+        std::cout << "Kernel Backward Time Elapsed: " << kernel_avg_time << " ms" << std::endl;
+    }
+
+    if(backward_output_dev->FromGPU(GetStream(), backward_output.data()) != 0)
+        std::cerr << "Error copying data from GPU, size: " << in_dev->GetSize() << std::endl;
+
+    return miopenStatusSuccess;
+}
+
+template <typename Tgpu, typename Tref>
+int ConstantPadDriver<Tgpu, Tref>::VerifyBackward()
+{
+    RunBackwardCPU();
+
+    // for (int i = 0; i < backward_output.size(); i++)
```

Comment:
remove comment

Reply:
I've removed the debug `printf` statement